### ISSUE TITLE
Wrong default value in GRU layer documentaion

### ISSUE THREAD

**Issue by rschumi0** [tensorflow/tensorflow#45705]
### Description of issue (what needs changing):
The default value for the recurrent_activation of the GRU layer does not seem to be sigmoid as it is specified in the documentation.

### Usage example
The following two code examples illustrate the issue. The first one specifies the recurrent_activation argument by setting it to 'sigmoid' and the second does not do that, but should have it as a default. However, the examples produce a slightly different result which suggests that there is another default value. (Tensorflow version  'v1.14.0-rc1-22-gaf24dc91b5', '1.14.0')
```
model = keras.Sequential([
keras.layers.LSTM(3,kernel_initializer='ones', recurrent_initializer='zeros',
    bias_initializer='zeros',  input_shape=(2,2),recurrent_activation='sigmoid')])
x= tf.constant([[[1, 2],[2,3]],[[1, 1],[0,1]],[[1, 2],[2,3]]])
print model.predict(x,steps=1)
```

```
model = keras.Sequential([
keras.layers.LSTM(3,kernel_initializer='ones', recurrent_initializer='zeros',
    bias_initializer='zeros',  input_shape=(2,2))])
x= tf.constant([[[1, 2],[2,3]],[[1, 1],[0,1]],[[1, 2],[2,3]]])
print model.predict(x,steps=1)
```

**Comment by amahendrakar** [comment]
@rschumi0,
TensorFlow 1.x is not actively supported. Please update TensorFlow to the latest stable version v2.4.

Using TensorFlow v2.4, I got similar results for both the cases. Please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/43411edfb227c92ac748314dc3280c80/45705.ipynb#scrollTo=OqpOs1vAvxzY). Thanks!
